usaspending_api/etl/management/commands/update_total_funding_amount_sql.py

```diff
@@ -0,0 +1,56 @@
+"""
+Creates legal entities and adds them to transaction normalized rows and award rows based on transaction_location_data
+from `create_locations`
+"""
+
+import logging
+
+from django.core.management.base import BaseCommand
+from django.db import connection
+
+
+logger = logging.getLogger('console')
+exception_logger = logging.getLogger("exceptions")
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
+
+        with connection.cursor() as curs:
+            curs.execute(self.TRANSACTION_FABS_TFA)
+            logger.info('total_funding_amount recalculated. {} successful updates'.format(curs.rowcount))
+
+            curs.execute(self.TRANSACTION_NORMALIZED_TFA)
+            logger.info('total_funding_amount moved to the transaction normalized table. {} successful updates'.format(
+                curs.rowcount))
+
+            curs.execute(self.SUM_TRANSACTION_TFA)
+            logger.info('New sum for total_funding_amount. {} successful updates'.format(curs.rowcount))
+
+    TRANSACTION_FABS_TFA = """
+        update transaction_fabs as tf
```

Comment:
I believe the field on `transaction_fabs` for `total_funding_amount` comes from FABS submissions. We shouldn't overwrite it here.